src/apis/drive/samples/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "google-drive-samples",
+  "version": "0.0.1",
+  "description": "These samples allow you to list, download, and upload files from Google Drive.",
+  "main": "quickstart.js",
+  "scripts": {
```

Comment:
meh, no need to have it

Reply:
it was autogen'd by `npm init`, will remove